models/model_runner.py

```diff
@@ -17,18 +17,38 @@ def get_args():
     parser.add_argument('--k_cluster', help='If running clustering on Data Processing')
     parser.add_argument('--capture_index', help='If running track and crop on Data Processing')
     parser.add_argument('--acc_threshold', help='If running track and crop on Data Processing')
+    parser.add_argument('--model_weights', help='The weights that should be used (for reID model)')
```

Comment:
having the model weight optional reverts to training from scratch or downloading / using some default checkpoint? maybe we can explain this in the args.